.github/ISSUE_TEMPLATE/es.md

```diff
@@ -3,20 +3,20 @@ name: Español
 about: Reportar un problema
 labels: l/es
 ---
-<!-- Utiliza esta plantilla para informar de incidencias y proporciona tanta información como sea posible. Si no lo haces, es posible que la respuesta se retrase. Muchas gracias. -->
+<!-- Utilize esta plantilla para informar de incidencias y proporciona tanta información como sea possible. Si no lo haces, es possible queue la respuesta se retrase. Muchas gracias. -->
```

Comment:
The issue template corrections for non-English languages appear to be incorrect. Please remove them from this PR.